src_py/sysfont.py

```diff
@@ -196,6 +198,9 @@ def initsysfonts_unix(path="fc-list"):
     """use the fc-list from fontconfig to get a list of fonts"""
     fonts = {}
 
+    if sys.platform == "emscripten":
```

Comment:
This makes the code run, which is good. 

I made an issue because this is silently disabled so it's good to track somewhere that we need this fixed/implemented: https://github.com/pygame/pygame/issues/3432